source/_components/enocean.markdown

```diff
@@ -95,9 +104,12 @@ device_class:
   type: device_class
 {% endconfiguration %}
 
-EnOcean binary sensors only generate 'button_pressed' events. The event data has following four fields:
+EnOcean binary sensors have no state, they only generate 'button_pressed' events. The event data has following fields:
 
 - **id**: The ID of the device (see configuration).
+- **id_str**: The ID of the device, formatted as colon-delimited string: `01:90:84:3C`.
```

Comment:
This wasn't implemented.

Reply:
Good catch, I reverted the event's sample.